components/JEDI/JEDINav.js

```diff
@@ -0,0 +1,41 @@
+import Link from 'next/link';
+import React from 'react';
+
+import { blogLink } from '../../data/jedi';
+import styles from '../../styles/pages/jedi.module.scss';
+
+const links = [
+  {
+    path: '/jedi',
+    title: 'home',
+  },
+  {
+    path: '/jedi/meet',
+    title: 'meet the JEDIs',
+  },
+  {
+    path: '/jedi/allyship',
+    title: 'allyship spaces',
+  },
+  {
+    path: blogLink,
+    title: 'blog',
+    ext: true,
```

Comment:
Can we add logic here where external links open in a new tag (i.e. `target="_blank"`)

Reply:
Hm, a bit confused - right now, the blog link already does this, what exactly are you looking for?

Reply:
When I click on the blog link on the index page, it doesn't open a new tab
![image](https://user-images.githubusercontent.com/23728476/132259514-03f4709c-3fb2-4c5d-b4ce-43d44d6a6d77.png)

However when I navigate through the horizontal nav bar it does the intended behavior
![image](https://user-images.githubusercontent.com/23728476/132259535-12533c80-b4e1-4b80-bc8b-59f3f70dd141.png)

Reply:
Ah makes perfect sense, my mistake. Will implement that!